src/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0-buster-slim AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0-buster AS build
+WORKDIR /src
+COPY ["/BlazorBoilerplate.Server/BlazorBoilerplate.Server.csproj", "BlazorBoilerplate.Server/"]
+COPY ["/BlazorBoilerplate.Shared/BlazorBoilerplate.Shared.csproj", "BlazorBoilerplate.Shared/"]
+COPY ["/BlazorBoilerplate.Client/BlazorBoilerplate.Client.csproj", "BlazorBoilerplate.Client/"]
+RUN dotnet restore "BlazorBoilerplate.Server/BlazorBoilerplate.Server.csproj"
+COPY . .
+WORKDIR "/src/BlazorBoilerplate.Server"
+RUN dotnet build "BlazorBoilerplate.Server.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "BlazorBoilerplate.Server.csproj" -c Release -o /app/publish
```

Comment:
please add the --no-restore flag to build command as the project has already been restored.
please add the --no-build flag to publish command as the project has been built before